packages/jest-cli/src/__tests__/watch-test.js

```diff
@@ -78,13 +77,39 @@ describe('Watch mode flows', () => {
 
     expect(runJestMock).toBeCalled();
     expect(argv).toEqual({
-      '_': '',
       onlyChanged: false,
       watch: false,
       watchAll: true,
     });
   });
 
+  it('Pressing "T" enters testname pattern mode', () => {
```

Comment:
Maybe we can update these tests to look similar to https://github.com/facebook/jest/blob/master/packages/jest-cli/src/__tests__/watch-pattern-mode-test.js#L86 in those tests we heavily use snapshots and they give us some nice insight of how the state updates over time :)